### DISCUSSION TITLE
هل ما زال المشروع حيًّا؟

### DISCUSSION THREAD

**Discussion by iTzVoko** [softvenue/i18n#89] [General]
السلام عليكم ورحمة الله وبركاته

يظهر لي أنَّ آخر التعديلات هي قبل سنتين فأكثر،
فهل تطوير المشروع واقف -لا سمح الله تعالى-؟ وما السبب؟

جزيل الشكر،
علي

**Comment by Aissaoui-Ahmed** [comment]
تفضل، أخي لا يزال على قيد الحياة. هل لديك اي اقتراحات؟

**Comment by forzagreen** [comment]
شكرا على مجهوداتكم.
للمهتمين، نعمل منذ سنوات على تجميع البيانات التقنية والعلمية في قاعدة بيانات:
- المشروع: https://github.com/forzagreen/arabterm
- لتصفح البيانات: https://forzagreen.github.io/arabterm
- محرك البحث: https://wikitermbase.toolforge.org/
